src/components/textbox/__next__/__internal__/error-border/error-border.style.ts

```diff
@@ -0,0 +1,18 @@
+import styled, { css } from "styled-components";
+
+const ErrorBorder = styled.span`
+  ${({ warning }: { warning: boolean }) => css`
+    width: 2px;
```

Comment:
comment: there are a couple of spacing tokens that have 2px for value, might be worht checking with DS/UX if any apply here